docker-bake.hcl

```diff
@@ -413,24 +385,27 @@ target "virtual-osbuild-ci-cXs" {
 }
 
 target "osbuild-ci-c9s-latest" {
-        args = {
-                OSB_FROM = "quay.io/centos/centos:stream9",
-        }
         inherits = [
                 "virtual-osbuild-ci-cXs",
         ]
+        args = {
+                // DNF package list need to be appended for c9s specific packages
+                OSB_DNF_PACKAGES = format("%s,%s", join(",", setsubtract(split("\n", BASE_PACKAGES), split("\n", CENTOS_REMOVED_PACKAGES))), "python3-tomli")
```

Comment:
Why is `python3-tomli` specified separately?

Reply:
This package is only presented on c9s and it is not presented on c10s. On c10s it use to be 'python3-tomli-w` before it has been removed. As this is only single package I have added it to the list by `format` rather than adding new set of variables. In future, if there will be more entries it will evolve into separated variable and dnf will receive `BASE_PACKAGES` - `CENTOS_REMOVED_PACKAGES` + `C9S_SPECIFIC_PACKAGES`